src/Microsoft.AspNetCore.Identity.Specification.Tests/TestRole.cs

```diff
@@ -35,6 +35,9 @@ public TestRole(string roleName) : this()
     /// <typeparam name="TKey"></typeparam>
     public class TestRole<TKey> where TKey : IEquatable<TKey>
     {
+        /// <summary>
```

Comment:
FWIW, I did not expect the specification package for providers to contain entity types because I thought each provider would bring their own. I rather expected the specification package to contain a set of base test classes which provider writers could derive from and template methods which provider writers could override/implement. 

Are we taking a different approach?